data_specs/Landsat_C2_SR_specs.ipynb

```diff
@@ -448,7 +448,48 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "**Table 9: AWS file path convention.**\n",
+    "### Amazon Simple Notification Service "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Landsat Collection 2 Level-2 stacs can be received thanks to the Public Dataset Program."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Table 9: AWS SNS access details.**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "|AWS Simple Notification Services details | |\n",
+    "|----------|-------------|\n",
+    "| SNS ARN | `arn:aws:sns:af-south-1:543785577597:deafrica-landsat-scene-topic` |\n",
+    "| Region |`af-south-1` |\n",
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The SNS endpoints are in the AWS region `af-south-1` (Cape Town). Additional region specifications can be applied as follows:\n",
+    "\n",
+    "`aws sns list-topics --region=af-south-1 `\n",
```

Comment:
This won't work for anyone unless they're on our account.

Reply:
removed